breez/c-breez
# Mnemonics enhancements

This PR addresses https://github.com/breez/c-breez/issues/474

- Remove MnemonicsVerificationHandler & VerifyMnemonicsDialog
  - Verify warning won't be shown after every payment and MnemonicsConfirmation page will be navigated directly from warning action
- Display MnemonicsPage in view mode if user has verified their mnemonics
- Persist verificationStatus
- Use new translated messages
